storage/src/main/java/zipkin2/storage/kafka/KafkaSpanConsumer.java

```diff
@@ -110,7 +89,7 @@
     protected Void doExecute() throws IOException {
       try {
```

Comment:
for parity you could consider a copy/pasta or even a dependency on this https://github.com/openzipkin/zipkin-reporter-java/blob/master/kafka/src/main/java/zipkin2/reporter/kafka/KafkaSender.java#L305

Reply:
I did both. Not able to reuse call as I need to add keys for partitioning, but copy paste useful callbacks :+1: